cloverage/src/cloverage/instrument.clj

```diff
@@ -332,19 +377,20 @@
 (defmethod do-wrap :new [f line [new-sym class-name & args :as form] _]
   (f line `(~new-sym ~class-name ~@(doall (map (wrapper f line) args)))))
 
-(defmethod do-wrap :dotjava [f line [dot-sym obj-name attr-name & args :as form] env]
-  ;; either (. obj meth args*) or (. obj (meth args*))
+(defmethod do-wrap :dotjava
+  [f line [dot-sym obj-name & more] env]
+  ;; either (. obj meth args*) or (. obj (meth args*)) -- syntaxes are equivalent.
   ;; I think we might have to not-wrap symbols here, or we might lose metadata
   ;; (like :tag type hints for reflection when resolving methods)
-  (if (= :list (form-type attr-name env))
-    (do
-      (d/tprnl "List dotform, recursing on" (rest attr-name))
-      (f line `(~dot-sym ~obj-name (~(first attr-name)
-                                    ~@(doall (map (wrapper f line)
-                                                  (rest attr-name)))))))
-    (do
-      (d/tprnl "Simple dotform, recursing on" args)
-      (f line `(~dot-sym ~obj-name ~attr-name ~@(doall (map (wrapper f line) args)))))))
+  (if (seq? (first more))
```

Comment:
#305 was caused by trying to determine the `form-type` of the java interop form. The intention here is to see whether we're using the `(. obj method & args)` or `(. obj (method & args))` syntax, but `form-type` could return something other than `:list` depending on what the method name was.

Using `seq?` here is sufficient.